addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -227,9 +231,27 @@ export class AddPageTemplatePreview extends Component {
                 holderEl.classList.add("o_ready");
             };
             adjustHeight();
+            if (this.props.isCustom) {
+                this.adaptCustom(wrapEl);
+            }
         });
     }
 
+    adaptCustom(wrapEl) {
+        for (const sectionEl of wrapEl.querySelectorAll("section:not(.o_snippet_desktop_invisible)")) {
+            const style = window.getComputedStyle(sectionEl);
+            if (!style.height || style.display === 'none') {
+                const messageEl = document.createElement("section");
+                messageEl.classList.add("o_new_page_snippet_preview", "container", "h2");
+                const divEl = document.createElement("div");
+                divEl.classList.add("alert", "alert-info");
+                divEl.textContent = _t("No preview for the %s block because it is dynamically rendered.", sectionEl.dataset.name);
```

Comment:
This line is too long :arrow_down: 
```js
divEl.textContent = _t(
    "No preview for the %s block because it is dynamically rendered.",
    sectionEl.dataset.name
);
```